/usr/bin/python3.9: can't open file '/home/ec2-user/supplychain_backendv2/app/manage.py': [Errno 2] No such file or directory
/usr/bin/python3.9: can't open file '/home/ec2-user/supplychain_backendv2/app/manage.py': [Errno 2] No such file or directory
/usr/bin/python3.9: can't open file '/home/ec2-user/supplychain_backendv2/app/manage.py': [Errno 2] No such file or directory
